spring-cloud-task-stream/src/main/java/org/springframework/cloud/task/launcher/TaskLauncherConfiguration.java

```diff
@@ -42,6 +44,9 @@
 @ConditionalOnClass({TaskLauncher.class})
 public class TaskLauncherConfiguration {
 
+	@Value("${maven.remoteRepositories.springRepo.url:https://repo.spring.io/libs-snapshot}")
+	private String repository;
```

Comment:
Why add this property when you could have just relied on the `MavenProperties` below which were exposed as boot properties? For one thing, if someone needs to set auth or proxy settings, they would need those as well, and now that you've removed `@ConfigurationProperties` there, those other properties are no longer available.